input/resources/library/Library-CQMCommonExample.json

```diff
@@ -78,11 +78,11 @@
   "relatedArtifact": [ {
     "type": "depends-on",
     "display": "QICore model information",
-    "resource": "http://hl7.org/fhir/Library/QICore-ModelInfo"
+    "resource": "http://hl7.org/fhir/us/qicore/Library/QICore-ModelInfo"
   }, {
     "type": "depends-on",
     "display": "Library FHIRHelpers",
-    "resource": "http://hl7.org/fhir/Library/FHIRHelpers|4.0.1"
+    "resource": "http://fhir.org/guides/cqf/common/Library-FHIRHelpers.html"
```

Comment:
This shouldn't have an .html in it? It's a canonical URL, not an actual web-address

Reply:
@brynrhodes So, the http://hl7.org/fhir/Library/FHIRHelpers|4.0.1 does not resolve, I have tried it without the version as well.